runtime/doc/treesitter.txt

```diff
@@ -400,58 +400,116 @@ instance, to highlight comments differently per language: >vim
+
+@function.method        method definitions
+@function.method.call   method calls
+
+@constructor            constructor calls and definitions
+@operator               symbolic operators (e.g. `+` / `*`)
+
+@keyword                keywords not fitting into specific categories
+@keyword.coroutine      keywords related to coroutines (e.g. `go` in Go, `async/await` in Python)
+@keyword.function       keywords that define a function (e.g. `func` in Go, `def` in Python)
+@keyword.operator       operators that are English words (e.g. `and` / `or`)
+@keyword.import         keywords for including modules (e.g. `import` / `from` in Python)
+@keyword.storage        modifiers that affect storage in memory or life-time
+@keyword.repeat         keywords related to loops (e.g. `for` / `while`)
+@keyword.return         keywords like `return` and `yield`
+@keyword.debug          keywords related to debugging
+@keyword.exception      keywords related to exceptions (e.g. `throw` / `catch`)
+
+@keyword.conditional         keywords related to conditionals (e.g. `if` / `else`)
+@keyword.conditional.ternary ternary operator (e.g. `?` / `:`)
+
+@keyword.directive           various preprocessor directives and shebangs
+@keyword.directive.define    preprocessor definition directives
+
+@punctuation.delimiter  delimiters (e.g. `;` / `.` / `,`)
+@punctuation.bracket    brackets (e.g. `()` / `{}` / `[]`)
+@punctuation.special    special symbols (e.g. `{}` in string interpolation)
+
+@comment                line and block comments
+@comment.documentation  comments documenting code
+
+@comment.error          error-type comments (e.g. `DEPRECATED:`)
+@comment.warning        warning-type comments (e.g. `WARNING:`, `FIX:`)
+@comment.hint           note-type comments (e.g. `NOTE:`)
+@comment.info           info-type comments
+@comment.todo           todo-type comments (e.g. `TODO:`, `WIP:`)
+
+@markup.strong          bold text
+@markup.italic          text with emphasis
+@markup.strikethrough   strikethrough text
+@markup.underline       underlined text (only for literal underline markup!)
+
+@markup.heading         headings, titles (including markers)
+
+@markup.quote           block quotes
+@markup.math            math environments (e.g. `$ ... $` in LaTeX)
+@markup.environment     environments (e.g. in LaTeX)
+
+@markup.link            text references, footnotes, citations, etc.
+@markup.link.label      link, reference descriptions
+@markup.link.url        URL-style links
+
+@markup.raw             literal or verbatim text (e.g. inline code)
+@markup.raw.block       literal or verbatim text as a stand-alone block
+
+@markup.list            list markers
+@markup.list.checked    checked todo-style list markers
+@markup.list.unchecked  unchecked todo-style list markers
+
+@diff.plus              added text (for diff files)
+@diff.minus             deleted text (for diff files)
+@diff.delta             changed text (for diff files)
+
+@tag                    XML tag names
```

Comment:
IDK how clear we were about this before, but what this makes me think as a vanilla Neovim user that has no advanced context/knowledge of these changes is: "Ok I don't need to worry about `@tag` in my color theme bc I don't use XML at all", yet a seemingly unrelated language like `yuck` [uses it in its queries](https://github.com/nvim-treesitter/nvim-treesitter/blob/4a4dbe1cb1da34d87fc42a40aaf8e218af4cfe0f/queries/yuck/highlights.scm#L4).

Maybe it would be worth it to document such a gotcha here?

Reply:
Shouldn't that be on yuck query maintainers to document that :? :thinking: I tried a 

```yuck
(button :onclick "notify-send Left"
  :onrightclick "notify-send Right"
  "Click Me!") ;; this is a label nested within a button
```
Where button is the `symbol`, and the syntax have some similarities to XML

Reply:
Fair point, but I'm bringing this up from the perspective of a Neovim user that's updating their TS queries to satisfy their colorful desires: If I find a query that at first glance look language/file extension exclusive, I might forget to consider which other languages might be affected.

I highlight these capture groups in particular because XML is included in the name, instead of an example as in previous groups.

Reply:
That's fine, feel free to suggest a better description. (I don't do weebdev.)

Reply:
We can clean up improperly used `@tag` highlights in nvim-treesitter later (probably should have handled it in my consistency PR tbh), but yuck (and other?) widgets are similar to XML tags and can maybe also use it.